onboarded.txt

```diff
@@ -37,3 +37,4 @@ tv2.dk,tv2.dk.securedrop.tor.onion
 aftenbladet.no,aftenbladet.securedrop.tor.onion
 www.sueddeutsche.de,sueddeutsche.securedrop.tor.onion
 noyb.eu,noyb.securedrop.tor.onion
+news.bloomberglaw.com,bloombergindustrygroup.securedrop.tor.onion
```

Comment:
Just noting the inconsistency in the URL there pointing to the old Bloomberg Law address, but I checked and that's the most up-to-date one available (looks like it hasn't been updated on their side yet). Doesn't affect the functionality here and there's no alternative for us to use, so going to proceed with an approve.